src/containers/AccountConnection.jsx

```diff
@@ -214,33 +232,10 @@ class AccountConnection extends Component {
               onCancel={() => this.cancel()}
             />
           </div>
-          <div className={styles['col-account-connection-data']}>
-            { description &&
-              <div>
-                <h4>{t('account.connection.data.service.description')}</h4>
-                <p>
-                  <ReactMarkdown
-                    source={
-                      t(description)
-                    }
-                    renderers={{Link: props => <a href={props.href} target='_blank'>{props.children}</a>}}
-                  />
-                </p>
-              </div>
-            }
-            <h4>{t('account.connection.data.title')}</h4>
-            {connector.dataType &&
-              <ul className={styles['col-account-connection-data-access']}>
-                {connector.dataType.map(data =>
-                  <DataItem
-                    dataType={data}
-                    hex={connector.color.hex}
-                  />
-                )}
-              </ul>}
-            {!connector.dataType &&
-              <p>{t('dataType.none', {name: connector.name})}</p>}
-          </div>
+          <AccountConnectionData
+            t={t}
```

Comment:
@CPatchane : didn’t get your point, I did what I saw in https://github.com/cozy/cozy-collect/blob/master/src/containers/AccountConnection.jsx#L261